tests/unit/tasks/workspaces/test_list.py

```diff
@@ -0,0 +1,75 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from typing import TYPE_CHECKING
+
+import pytest
+
+if TYPE_CHECKING:
+    from click.testing import CliRunner
+    from pytest_mock import MockerFixture
+    from typer import Typer
+
+from argilla.tasks.workspaces.list import list_workspaces
+
+from tests.factories import WorkspaceFactory
+from tests.integration.conftest import *  # for owner fixture
```

Comment:
I would say to keep the integration and unit tests group. If you need an owner user, I recommend you  create it as a fixture in the context of the test

Reply:
It would also be nice to have tests for other kind of user roles.

Reply:
> I would say to keep the integration and unit tests group. If you need an owner user, I recommend you create it as a fixture in the context of the test

Sure, I should have done it the first time. @gabrielmbmb did the modifications and those fixtures are no longer needed.

Reply:
> It would also be nice to have tests for other kind of user roles.

With the last updates there is no explicit use of the user roles, would this still be needed?